tc-health-client/tmagent/tmagent.go

```diff
@@ -49,6 +49,14 @@ const (
 	StrategiesFile = "strategies.yaml"
 )
 
+// this global is used to auto select the
+// proper ATS traffic_ctl command to use
+// when querying host status. for ATS
+// version 10 and greater this will remain
+// at 0.  For ATS version 9, this will be
+// auto updated to 1
+var traffic_ctl_index = 0
```

Comment:
Nitpick: Go symbols should be consistentCased, i.e. `var trafficCtlIndex = 0`.
https://go.dev/doc/effective_go#mixed-caps

It'd also be ideal to avoid global variables, and pass it locally instead. But for what's effectively a singleton in a small app, not a huge deal